mesonbuild/compilers/c.py

```diff
@@ -1217,9 +1217,13 @@ def __init__(self, exelist, version, compiler_type, is_cross, exe_wrapper=None,
 
     def get_options(self):
         opts = CCompiler.get_options(self)
+        c_stds = ['c89', 'c99', 'c11']
+        g_stds = ['gnu89', 'gnu99', 'gnu11']
+        if version_compare(self.version, '>=7.0.0'):
```

Comment:
I don't think this is strictly correct, since apple clang doesn't support this till 10.0.0.  think this should be something like:
```python
if self.complier_type is CompilerType.CLANG_OSX:
    v = '>= 10.0.0'
else:
    v = '>=7.0.0'
if version_compare(self.version, v):
   ...
```

Reply:
Thanks, that's just the information I was looking for. I tested the change on my local system and then made it online to c.py. I also updated the release note snippet.

Reply:
Do I resolve the conversation, or do you do that?

Reply:
Either way.